src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -173,7 +175,9 @@ private Content transformEomFileToContent(UUID uuid, ParsedEomFile eomFile, Tran
 
         final String standfirst = Strings.nullToEmpty(xpath.evaluate(STANDFIRST_XPATH, value)).trim();
 
-        final String transformedBody = transformField(eomFile.getBody(), bodyTransformer, transactionId);
+        final String transformedBody = transformField(eomFile.getBody(), bodyTransformer, transactionId,
+            Maps.immutableEntry("uuid", uuid.toString())
```

Comment:
should we have the key (`uuid`) extracted to a constant somewhere? I'm leaning towards a yes